src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -29,6 +35,7 @@
     <UpdateDependencies DependencyBuildInfo="@(DependencyBuildInfo)"
                         ProjectJsonFiles="@(ProjectJsonFiles)"
                         XmlUpdateStep="@(XmlUpdateStep)"
+                        BuildInfoCacheDir="$(BuildInfoCacheDir)"
```

Comment:
The cache is for buildinfo data downloaded from the versions repo--`@(DependencyBuildInfo)` has pointers to where in the versions repo to look. This is what Wes and I talked about here: https://github.com/dotnet/buildtools/issues/732#issuecomment-220122400